katalogus/views/plugin_enable_disable.py

```diff
@@ -58,6 +58,9 @@ def post(self, request, *args, **kwargs):
                 )
         return HttpResponseRedirect(request.POST.get("current_url"))
 
+    def add_info_message(self, message):
+        messages.add_message(self.request, messages.INFO, message)
```

Comment:
This seems a bit redundant if it's only referenced once. Any reasons not to inline this call?

Reply:
Just wanted to stay in line with the other "add_xxxxx_message()" methods. But the same can be said for those, so I will clean them up, because I agree :)